src/main/java/taxi/dao/DriverDaoImpl.java

```diff
@@ -74,12 +76,15 @@ public Driver create(Driver driver) {
     public Driver update(Driver driver) {
         String query = "UPDATE drivers "
                 + "SET name = ?, license_number = ? "
+                + "SET login = ?, password = ? "
```

Comment:
Does it work correctly?
![image](https://user-images.githubusercontent.com/94574503/161577419-abc4a7c2-d12f-4284-9a88-8073fbeacc00.png)